config.json

```diff
@@ -890,11 +890,12 @@
       "slug": "change",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 3,
       "topics": [
```

Comment:
Consider adding "Searching", since the goal is to search for the smallest set of coins based on the input set.